File: src/sdk-reference/cpp/1/document/validate/snippets/validate.cpp
try {
  bool valid = kuzzle->document->validate("nyc-open-data", "yellow-taxi", R"({
    "capacity": 4
  })");

  if (valid) {
    std::cout << "The document is valid" << std::endl;
  }
} catch (kuzzleio::KuzzleException& e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk/java/1/controllers/document/validate/snippets/validate.java
String doc = "{\"capacity\": 4}";

try {
   boolean valid = kuzzle.getDocument().validate("nyc-open-data", "yellow-taxi", doc);

    if (valid) {
      System.out.println("Success");
    }
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}
